src/content/docs/en/recipes/add-yaml-support.mdx

```diff
@@ -34,3 +34,15 @@ Astro builds on top of Vite, and supports both Vite and Rollup plugins. This rec
     ```js
     import yml from './data.yml';
     ```
+
+    :::note
+    While you can now import YAML data in your Astro project, your editor will not provide types for the imported data. To add types, create a `*.d.ts` file in the `src` directory of your project and add the following:
```

Comment:
```suggestion
    While you can now import YAML data in your Astro project, your editor will not provide types for the imported data. To add types, create or find an existing `*.d.ts` file in the `src` directory of your project and add the following:
```

Also just to check:
- Must this go in its own, new file? Can it go in an existing file along with other things? If so, I might suggest something like above.

Reply:
Yep, an existing file is fine!